### DISCUSSION TITLE
liqoctl plugin for asdf

### DISCUSSION THREAD

**Discussion by pdemagny** [liqotech/liqo#1546] [Ideas]
Hi,

I made a plugin for [asdf](https://github.com/asdf-vm/asdf) to install liqoctl easily with this great tool.
I would like to know if:
1. You are ok with this, my repo can stay public, and you give me the permissions to make a PR to make my plugin (which is based on the official asdf plugin template) a permanent plugin of asdf by creating a PR on [the asdf plugins repo](https://github.com/asdf-vm/asdf-plugins). I could also make a PR in this repo to update the installation docs with this new installation method.
2. You are not ok with this and I shall make my repo private and not submit it to the asdf plugins repo !

Please tell me what you think ;)

**Comment by pdemagny** [comment]
I forgot the link to my plugin's repo: [asdf-liqoctl](https://github.com/pdemagny/asdf-liqoctl)

**Comment by giorio94** [comment]
Hi @pdemagny,

Thanks! I personally didn't know about *asdf*, but it seems to me this could be great addition to liqo.
From my side, feel free to keep it public and go on with the PR to their repository. 
It would be much appreciated if you could also make the PR to update the liqoctl installation docs with this new method.

Just to know, will any action be required on the liqo side when a new version is released? Or is it all automatic?

**Reply by pdemagny** [reply]
Hi @giorio94,

Thanks for your quick and positive reply ;)
I'll proceed to do both PRs then, including the one for liqoctl installation docs !

Normally it is all automatic and as long as you don't change the name of the liqoctl releases files in Github, it would still work and auto-discover new versions :+1:

**Reply by giorio94** [reply]
Thanks! Great to hear that!
